benches/schema_validation_benchmark.rs

```diff
@@ -0,0 +1,886 @@
+use criterion::{criterion_group, criterion_main, Criterion};
+use regorus::Value;
+use regorus::{Schema, SchemaValidator};
+use serde_json::json;
+
+// Observed: validate_string - 3.19 ns/iter
```

Comment:
>3.19 ns/iter [](http://example.com/codeflow?start=31&length=12)

Just curious, does it make sense to include these? Don't they rely on the hardware used? (I don't mind these being here, just curious. One reason to keep them is to get an idea of order of magnitude difference between individual methods.)